fe/fe-core/src/main/java/com/starrocks/sql/optimizer/task/OptimizeGroupTask.java

```diff
@@ -36,12 +35,12 @@ public void execute() {
             return;
         }
 
-        for (GroupExpression expression : group.getLogicalExpressions()) {
-            pushTask(new OptimizeExpressionTask(context, expression));
+        for (int i = group.getLogicalExpressions().size() - 1; i >= 0; i--) {
+            pushTask(new OptimizeExpressionTask(context, group.getLogicalExpressions().get(i)));
```

Comment:
Why need to traverse in reverse order?

Reply:
just for keep enforce logical plan order